python/cuml/_thirdparty/sklearn/preprocessing/_imputation.py

```diff
@@ -232,6 +241,12 @@ def __init__(self, *, missing_values=np.nan, strategy="mean",
         self.verbose = verbose
         self.copy = copy
 
+    def get_param_names(self):
+        return super().get_param_names() + [
+            "statistics_",
+            "features_",
+        ]
+
```

Comment:
Estimated properties should not be returned in `get_param_names()`. You want to return the arguments used in `__init__`. For this class that would be `strategy`, `fill_value`, `verbose`, and `copy`.

Reply:
True, I went a little fast on this one. This function should list the attributes that are necessary for the duplication/pickling of the estimator, right? I guess, it's the arguments used in `__init__`, but also the attributes that were acquired during training and that would be used when calling the `transform` method. I'll fix that.